### PR TITLE
DBZ-374 Demo for CDC unwrap

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: unwrap-smt/README.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  * Kafka Broker
+  * Kafka Connect with [Debezium](http://debezium.io/) and  [JDBC](https://github.com/confluentinc/kafka-connect-jdbc) Connectors
+* PostgreSQL
+
+## Usage
+How to run:
+
+```shell
+# Start the application
+DEBEZIUM_VERSION=0.6
```
Comment (gunnarmorling) [inline unwrap-smt/README.md]: `export` missing?

File: unwrap-smt/source.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        "database.hostname": "mysql",
+        "database.port": "3306",
+        "database.user": "debezium",
+        "database.password": "dbz",
+        "database.server.id": "184054",
+        "database.server.name": "dbserver1",
+        "database.whitelist": "inventory",
+        "database.history.kafka.bootstrap.servers": "kafka:9092",
+        "database.history.kafka.topic": "schema-changes.inventory",
+        "transforms": "route",
```
Comment (gunnarmorling) [inline unwrap-smt/source.json]: Do we need the router for this demo?

File: unwrap-smt/README.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+ Bailey    | 1002 | George     | gbailey@foobar.com
+ Walker    | 1003 | Edward     | ed@walker.com
+ Kretchmar | 1004 | Anne       | annek@noanswer.org
+(4 rows)
+```
+### New record
+Insert a new record into MySQL
+```shell
+docker-compose exec mysql bash -c 'mysql -u $MYSQL_USER  -p$MYSQL_PASSWORD inventory'
+mysql> insert into customers values(default, 'Reinhard', 'Scheer', 'cic@hsf.de');
```
Comment (gunnarmorling) [inline unwrap-smt/README.md]: Who is this dude ;)

File: unwrap-smt/README.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+                          |
+                  +-------v--------+
+                  |                |
+                  |   PostgreSQL   |
+                  |                |
+                  +----------------+
+
+
+```
+We are using a Docker Compose to deploy following components
```
Comment (gunnarmorling) [inline unwrap-smt/README.md]: s/a Docker/Docker/

File: unwrap-smt/README.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+                   |    MySQL    |
+                   |             |
+                   +------+------+
+                          |
+                          |
+                          |
+          +---------------v------------------+
+          |                                  |
+          |           Kafka Connect          |
+          |  (Debezium, JDBC, ElasticSearch  |
```
Comment (gunnarmorling) [inline unwrap-smt/README.md]: Elasticsearch shouldn't be part of this.

File: unwrap-smt/README.md (hunk 1 above)
Comment (jpechane) [reply unwrap-smt/README.md]: I do not think it is necessary but I've added it.

File: unwrap-smt/source.json (hunk 2 above)
Comment (jpechane) [reply unwrap-smt/source.json]: DBZ topic naming is not compatible with JDBC sink naming - more to be explained in following blogpost.

File: unwrap-smt/README.md (hunk 3 above)
Comment (jpechane) [reply unwrap-smt/README.md]: Let's say that I've an unusual hobby ;-) https://de.wikipedia.org/wiki/Skagerrakschlacht

**Comment by gunnarmorling** [comment]
Squashed and merged. Thanks, @jpechane!

**Pull request merged**